lib/matplotlib/tests/test_colors.py

```diff
@@ -222,6 +223,13 @@ def test_SymLogNorm():
     norm = mcolors.SymLogNorm(3, vmin=-30, vmax=5, linscale=1.2)
     normed_vals = norm(vals)
     assert_array_almost_equal(normed_vals, expected)
+    
+    # Ensure that a SymLogNorm object can be used in a colorbar
```

Comment:
This should be it's own test (any function name that starts with `test_` will work).

It also needs the `@cleanup` decorator.